### PR TITLE
send current line

### PR DESCRIPTION
**Pull request by kindlychung**
Added a new function and a corresponding command to send the current line to repl.

### CODE REVIEW

File: plugin/slime.vim [hunk 1]
```diff
@@ -277,6 +282,7 @@ command -bar -nargs=0 SlimeConfig call s:SlimeConfig()
 command -range -bar -nargs=0 SlimeSend <line1>,<line2>call s:SlimeSendRange()
 command -nargs=+ SlimeSend1 call s:SlimeSend(<q-args> . "\r")
 command -nargs=+ SlimeSend0 call s:SlimeSend(<args>)
+command! SlimeSendCL call s:SlimeSendCurrentLine()
```
Comment (jpalardy) [inline plugin/slime.vim]: How about inlining the whole line here?

    command! SlimeSendCurrentLine call s:SlimeSend(getline(".") . "\r")

**Comment by jpalardy** [comment]
How do you use this in practice?
Do you have a key binding to invoke this?
Is it better than `shift-v ctrl-c, ctrl-c`?

File: plugin/slime.vim (hunk 1 above)
Comment (kindlychung) [inline plugin/slime.vim]: That's also fine with me.

**Comment by kindlychung** [comment]
I mapped it to <c-c><c-l>, and yes, the experience is significantly better.

File: plugin/slime.vim (hunk 1 above)
Comment (jpalardy) [inline plugin/slime.vim]: If you could update the PR, then I'll merge it.

**Comment by jpalardy** [comment]
great, thanks :-)

**Pull request merged**